appveyor.yml

```diff
@@ -27,10 +27,12 @@ environment:
     # - TOOLCHAIN: "vs-14-2015"
     #   PROJECT_DIR: examples\ogles_gpgpu
 
-    - TOOLCHAIN: "vs-14-2015-win64-sdk-8-1"
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
```

Comment:
It seems to work without explicit `APPVEYOR_BUILD_WORKER_IMAGE`.

Configuration:
* https://github.com/ingenue/hunter/blob/a4450c38c4475424cc56fb425513d43c48d3b992/appveyor.yml#L29-L35

Build:
* https://ci.appveyor.com/project/ingenue/hunter/build/1.0.3420

Reply:
Here's the appveyor output for the commit prior to adding `APPVEYOR_BUILD_WORKER_IMAGE`:

https://ci.appveyor.com/project/headupinclouds/hunter-8w0rs/build/1.0.21/job/0gu76d4nypmn7bls

In either case, I think a more explicit specification could be more invariant to changes in Appveyor/Travis default settings (it can't hurt).

Reply:
> Here's the appveyor output for the commit prior to adding APPVEYOR_BUILD_WORKER_IMAGE

Looks strange, may be it's an AppVeyor glitch? What if you restart job again? Have you done any customization in project settings?

> In either case, I think a more explicit specification could be more invariant to changes in Appveyor/Travis default settings (it can't hurt).

Ok.

Reply:
> What if you restart job again?

Same: https://ci.appveyor.com/project/headupinclouds/hunter-8w0rs

> Have you done any customization in project settings?

I don't recall modifying anything.  Maybe it just has old defaults?  I see

```
appveyor -> hunter -> environment -> build worker image = visual studio 2013
```

Reply:
Although that also seems to be the default for a different project that was added recently.

Reply:
> Maybe it just has old defaults? I see

Yeah, I guess that's the reason, I got `Visual Studio 2015` there.

I'm planning to do mass toolchains update soon, will take this into account.